src/plot/voronoi.js

```diff
@@ -28,7 +29,7 @@ function Voronoi({
     <g className={`${className} rv-voronoi`} style={style}>
       {voronoiInstance.polygons(nodes).map((d, i) => (
         <path
-          className="rv-voronoi__cell"
+          className={`rv-voronoi__cell ${cellClassName(d.data)}`}
```

Comment:
I'm not a huge fan of the name `cellClassName` and perhaps it should also allow strings (not just functions). But I didn't want to make it more complex.